department-of-veterans-affairs/abd-vro
# Removes Single Contention Classification Endpoint

## What was the problem?
This PR cleans up the classification service by removing a deprecated end point and tests associated with this classifier.  This endpoint is being removed from vets-api codebase in [PR #18915](https://github.com/department-of-veterans-affairs/vets-api/pull/18915). 

Associated tickets or Slack threads:
- [#549](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/549)

## How does this fix it?[^1]
This removes the PR, updates test for the multi-classification service, and removes tests no longer applicable. 

## How to test this PR
- from `domain-cc` directory run `pytest`



[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
